### ISSUE TITLE
use_tidy_labels() issues with preexisting labels

### ISSUE THREAD

**Issue by DavisVaughan** [r-lib/usethis#1087]
yardstick has pre-existing issues labels (like `enhancement`). I went to run `use_tidy_labels()` on it and got back:

```r
> usethis::use_tidy_labels()
✓ Renaming labels: 'enhancement' -> 'feature'
  'good first issue' -> 'good first issue :heart:'
  'help wanted' -> 'help wanted :heart:'
  'question' -> 'reprex'
Error in out[[1]] : subscript out of bounds
```

I think this is related to the following issue, which has been fixed in the cran version of gh (the fix there is what causes the issue here)
https://github.com/r-lib/gh/issues/66

This map is what fails
https://github.com/r-lib/usethis/blob/54659c36c5391396535098209dd1e7ee6f25840a/R/github-labels.R#L110-L113

For yardstick, the `enhancement` label needs to be renamed but there are currently no issues with that label, so an empty response comes back. That means this branch of the usethis `gh()` function is run

https://github.com/r-lib/usethis/blob/54659c36c5391396535098209dd1e7ee6f25840a/R/github-labels.R#L84-L88

But since empty responses have changed in gh, the `out[[1]]` fails

**Comment by jimhester** [comment]
I just ran into this myself

**Comment by jennybc** [comment]
I think this was also fixed by #1088. Someone correct me if I'm wrong.
